kitsune/search/v2/base.py

```diff
@@ -9,8 +9,12 @@ class SumoDocument(DSLDocument):
     indexed_on = field.Date()
 
     @classmethod
-    def prepare(cls, instance):
-        """Prepare an object given a model instance"""
+    def prepare(cls, instance, merge_docs=False):
```

Comment:
To make this a bit more generic, I think we should add a `parent=None` argument, so as to not tie this to the KB's model layout. Indeed, we don't need `merge_docs` any more then, since all the conditionals that are `if merge_docs and instance.parent` can just be `if parent`.

Reply:
This would mean though that you will be merging docs every time you have a parent object which may not be what we want I don't see any other use case to be honest but I prefer a toggle for the functionality and not to base the outcome of the prepare method in the existence of an object.